runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectGraphVisitor.java

```diff
@@ -56,7 +61,7 @@ public CompositeBehavior enterCompositeTransform(TransformHierarchy.Node node) {
         "Attempting to traverse a pipeline (node %s) with a %s "
             + "which has already visited a Pipeline and is finalized",
         node.getFullName(),
-        ConsumerTrackingPipelineVisitor.class.getSimpleName());
+        DirectGraphVisitor.class.getSimpleName());
```

Comment:
Since you touched this line, good practice for non-final non-static methods to just use `getClass()`.